Library/ViewModels/PledgeViewModel.swift

```diff
@@ -66,15 +66,20 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
           .materialize()
       }
 
-    self.presentShippingRules = shippingRulesEvent.values()
-
-    let selectedShippingRule = Signal.merge(
+    let defaultShippingRule = Signal.merge(
       self.viewDidLoadProperty.signal.mapConst(nil),
-      self.presentShippingRules.map(defaultShippingRule(fromShippingRules:))
+      shippingRulesEvent.values().map(defaultShippingRule(fromShippingRules:))
+    )
+
+    self.presentShippingRules = Signal.combineLatest(
+      project,
+      shippingRulesEvent.values(),
+      defaultShippingRule.skipNil()
     )
+    .takeWhen(self.shippingRuleSignal.signal)
 
     let shippingAmount = Signal.merge(
-      selectedShippingRule.skipNil().map { $0.cost },
+      defaultShippingRule.skipNil().map { $0.cost },
```

Comment:
Renaming as this is the `default` value..when we allow selection we will add another signal replacing this.